C/CeresSolver/build_tarballs.jl

```diff
@@ -1,55 +1,32 @@
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
 name = "CeresSolver"
-version = v"1.14.0"
+version = v"2.1.0"
 
 sources = [
-    ArchiveSource("http://ceres-solver.org/ceres-solver-$(version).tar.gz",
-                  "4744005fc3b902fed886ea418df70690caa8e2ff6b5a90f3dd88a3d291ef8e8e")
+    GitSource("https://github.com/ceres-solver/ceres-solver.git",
+              "f68321e7de8929fbcdb95dd42877531e64f72f66")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/ceres-solver-1.14.0
-
-mkdir cmake_build
-cd cmake_build/
-
-if [[ "$target" == *-freebsd* || "$target" == *-apple-* ]]; then
-  # Clang doesn't play nicely with OpenMP and
-  # compilation fails with glog due to a c++11 error
-  CMAKE_FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake
-               -DMINIGLOG=ON)
-else
-  CMAKE_FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
-               -DGLOG_INCLUDE_DIR_HINTS=${prefix}/include)
-fi
-
 CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix}
+              -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+              -DCMAKE_BUILD_TYPE=Release
               -DBUILD_SHARED_LIBS=ON
               -DBUILD_EXAMPLES=OFF
               -DBUILD_TESTING=OFF
-              -DOPENMP=ON
-              -DTBB=OFF
               -DBLAS_LIBRARIES=${libdir}/libopenblas.${dlext}
               -DLAPACK_LIBRARIES=${libdir}/libopenblas.${dlext}
               -DMETIS_LIBRARY=${libdir}/libmetis.${dlext}
-              -DSUITESPARSE_INCLUDE_DIR_HINTS=${prefix}/include
-              -DAMD_LIBRARY="${libdir}/libamd.${dlext} ${libdir}/libsuitesparseconfig.${dlext}"
-              -DCAMD_LIBRARY="${libdir}/libcamd.${dlext} ${libdir}/libsuitesparseconfig.${dlext}"
-              -DCCOLAMD_LIBRARY="${libdir}/libccolamd.${dlext} ${libdir}/libsuitesparseconfig.${dlext}"
-              -DCHOLMOD_LIBRARY="${libdir}/libcholmod.${dlext} ${libdir}/libsuitesparseconfig.${dlext}"
-              -DCOLAMD_LIBRARY="${libdir}/libcolamd.${dlext} ${libdir}/libsuitepsarseconfig.${dlext}"
-              -DSUITESPARSEQR_LIBRARY="${libdir}/libspqr.${dlext} ${libdir}/libsuitesparseconfig.${dlext}"
-              -DSUITESPARSE_CONFIG_LIBRARY="${libdir}/libsuitesparseconfig.${dlext}"
               )
 
-cmake ${CMAKE_FLAGS[@]} ..
-
+cd $WORKSPACE/srcdir/ceres-solver/
+mkdir build && cd build
+export OPENBLAS_NUM_THREADS=1
```

Comment:
Why do you need to set this during the build? Looks weird.

Reply:
I saw that on the BLAS section here: http://ceres-solver.org/installation.html

Reply:
I doubt that they're talking about the build. That's probably a comment about runtime.

Reply:
That makes sense. I just removed that line.